packages/fx-core/resource/package.nls.json

```diff
@@ -252,7 +253,7 @@
   "plugins.cicd.whichEnvironment.title": "Select an environment",
   "plugins.cicd.ProgressBar.scaffold.title": "Scaffolding workflow automation files",
   "plugins.cicd.ProgressBar.scaffold.detail": "Scaffolding workflow file for %s.",
-  "plugins.cicd.result.scaffold.created": "Workflow automation - including %s using %s for %s environment - has been successfully added to your project. Follow the instructions in README file to configure your workflow(s).",
+  "plugins.cicd.result.scaffold.created": "Workflow automation - including %s using %s for %s environment - has been successfully added to your project. To configure your workflow(s) please follow the instructions in ",
```

Comment:
It will be hard for localization team to translate the string if we separate them without placeholder.

Reply:
Thank you, updated codes as discussed.